kurtosis_package/dockerfiles/contract_deployer.Dockerfile

```
@@ -1,46 +1,40 @@
-FROM debian:bookworm-slim AS foundry
+FROM debian:bookworm-slim as foundry
```

We can remove the stage name since it's no longer multi-stage.

```suggestion
FROM debian:bookworm-slim
```